### PR TITLE
More tiny fixes

### PR DESCRIPTION
**Pull request by marshallpierce**

### CODE REVIEW

File: third_party/rust-on-exercism/health-statistics.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 You're working on implementing a health-monitoring system. As part of that, you
 need to keep track of users' health statistics.
 
-You'll start with some stubbed functions in an `impl` block as well as a `User`
-struct definition. Your goal is to implement the stubbed out methods on the
+You'll start with a stubbed function in an `impl` block as well as a `User`
```
Comment (marshallpierce) [inline third_party/rust-on-exercism/health-statistics.md:4]: Juuuust one

File: src/borrowing/exercise.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ minutes: 30
 {{#include ../../third_party/rust-on-exercism/health-statistics.md}}
 
 Copy the code below to <https://play.rust-lang.org/> and fill in the missing
-methods:
+method:
```
Comment (marshallpierce) [inline src/borrowing/exercise.md:10]: There's only one method

File: src/slices-and-lifetimes/exercise.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ What remains for you is to implement the `parse_field` function.
 
 
 {{#include exercise.rs:parse_field }}
-    // 1. Read and unpack the tag.
+    // 1. Read and unpack the tag, which is a varint.
```
Comment (marshallpierce) [inline src/slices-and-lifetimes/exercise.md:50]: Students noted confusion over how to get the tag, and this detail _is_ present in the docs, but not obvious, so this seems like a fair hint

**Review by djmitche — COMMENTED** [review]
Thank you!!

**Review by djmitche — APPROVED** [review]
.

**Pull request merged**